config/routes.rb

```diff
@@ -28,13 +28,19 @@ def add_swagger_route http_method, path, opts = {}
   end
 
   add_swagger_route 'POST', '/stages/{stage_id}/actions', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'add_action'
+  add_swagger_route 'POST', '/requests/{request_id}/actions', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'add_action_by_request_id'
+  add_swagger_route 'POST', '/approvers', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'add_approver'
   add_swagger_route 'POST', '/groups', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'add_group'
   add_swagger_route 'POST', '/templates/{template_id}/workflows', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'add_workflow'
   add_swagger_route 'GET', '/actions/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_action_by_id'
   add_swagger_route 'GET', '/actions', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_actions'
+  add_swagger_route 'GET', '/approvers/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_approver_by_id'
+  add_swagger_route 'GET', '/approvers', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_approvers'
   add_swagger_route 'GET', '/groups/{id}', controller_name: 'admins', constraint_name: 'adminsConstraint', action_name: 'fetch_group_by_id'
```

Comment:
we should have `GET '/groups/{id}/approvers'`

Reply:
What are the endpoints to add/remove an approver to/from a group?

Reply:
Removing an approver from group can share the endpoint of `DELETE '/approvers/{id}'`, because `approver_id` is required in deletion. Instead, we may need an endpoint to delete all approvers from a certain group.

Reply:
We were talking about join/withdraw approvers from a group. It is resolved now.